app/models/state_file_base_intake.rb

```diff
@@ -202,6 +202,10 @@ def filer_count
     filing_status_mfj? ? 2 : 1
   end
 
+  def household_count
+    filer_count + dependents.size
+  end
```

Comment:
mike is implementing this on another PR but don't know who will get to merge it first